### PR TITLE
Use File scoped namespace statements

### PR DESCRIPTION
**Pull request by mic-max**
Fixes [#276](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/276)

To review please enable the ignore whitespace changes option
image

### CODE REVIEW

**Comment by reyang** [comment]
That's a huge PR! 😄

File: src/OpenTelemetry.Instrumentation.Wcf/TelemetryServiceBehavior.cs [hunk 1]
```diff
@@ -18,37 +18,46 @@
 using System.ServiceModel.Description;
 using System.ServiceModel.Dispatcher;
 
+
```
Comment (mic-max) [inline src/OpenTelemetry.Instrumentation.Wcf/TelemetryServiceBehavior.cs:21]: What's this...

**Review by reyang — APPROVED** [review]
LGTM, it's a big PR and the actual change will probably make `git blame` experience much harder.

Personally, I don't have problem with `git blame`, it's just an extra 30 seconds digging into the history. Not sure if the code owners found it problematic.

**Review by utpilla — APPROVED** [review]

**Comment by cijothomas** [comment]
@mic-max Has many conflicts.. could you resolve?

**Review by cijothomas — CHANGES_REQUESTED** [review]
Please submit smaller PRs. Its hard to look at 200+ files to make sure no accidental changes are made part of this PR. I get that the actual change is minimal, but need to ensure no accidental changes are part of this. I'd suggest to do at max ~25 files

**Pull request closed without merging**